### ISSUE TITLE
AdjustmentFactors variable names do not match SAM

### ISSUE THREAD

**Issue by dguittet** [NREL/pysam#164]
The Adjustment Factor variables in the SAM GUI have the prefix `adjust_` whereas the PySAM versions do not, as shown for Pvwattsv8 here: https://nrel-pysam.readthedocs.io/en/main/modules/Pvwattsv8.html#adjustmentfactors-group

This discrepancy will lead to issues reading inputs that were exported from the SAM GUI.

The workaround now is to modify the variable name:

```
for k, v in pv_inputs.items():
    if 'adjust_' in k:
         k = k.replace('adjust_', '')
```

**Comment by dguittet** [comment]
Fix in 9ded981f30c24751688155c93c73cebe77aa0753

**Comment by dguittet** [comment]
Previous fix of `k = k.split('adjust_')[1]` does not work when there are multiple types of adjustment factors (such as sf and dc)
 
The correct fix is
 
`k = k.replace('adjust_', '')`
